vllm/model_executor/models/bert.py

```diff
@@ -216,11 +215,6 @@ def __init__(
                               quant_config=quant_config,
                               prefix=f"{prefix}.attn")
 
-        if not isinstance(self.attn.impl, XFormersImpl):
-            raise ValueError(
-                "Encoder-only models currently require XFORMERS attention "
-                "backend. Set VLLM_ATTENTION_BACKEND=XFORMERS to use BERT.")
```

Comment:
Accordingly, we should remove the env override in `tests/models/embedding/language/test_embedding.py` since it's no longer necessary.